provider/etcd/etcd_client.go

```diff
@@ -269,9 +322,10 @@ func (ec *EtcdClient) RunLoadAllAlerts(ctx context.Context) {
 				if err != nil {
 					continue // retry
```

Comment:
MAY output some warning so that people know there is a problem when they check logs, maybe just the raw key/value from etcd.